fe49-onl-maryia-dauhan/src/App.tsx

```diff
@@ -1,24 +1,67 @@
-import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import { Button } from './ui/button/button';
+import { Title } from './ui/title/title';
+import { Header } from './features/header/header';
+import { TabDefault } from './features/tab/tabs';
+import { Input } from './ui/input/input';
+import { Textarea } from './ui/textarea/textarea';
+import { Listposts } from './pages/list-of-posts';
 
 function App() {
+  const onClick = () => console.log('Button clicked!');
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <Header />
+
+      <Title>Lesson task #37 and Hometask #37</Title>
+
+      <Title>Sign In</Title>
```

Comment:
Multiple h1 tags on the page is the semantic problem :(

Reply:
fixed